packages/flutter/lib/src/services/text_input.dart

```diff
@@ -721,10 +721,27 @@ TextAffinity? _toTextAffinity(String? affinity) {
   return null;
 }
 
-/// A floating cursor state the user has induced by force pressing an iOS
-/// keyboard.
+/// The state of a floating cursor drag on an iOS soft keyboard.
+///
+/// The floating cursor is an iOS feature in which long pressing and dragging on
+/// the spacebar of the soft keyboard causes the cursor in the text field to
+/// move along with the drag gesture. It's used to precisely position the cursor
+/// in some editable text.
+///
+/// This feature works out-of-the-box with Flutter. Support is built into
+/// [EditableText].
+///
+/// ![Animation of the native iOS floating cursor feature on an iPhone 14 simulator running iOS 17.0](
```

Comment:
This is the title of the image, but what we're supposed to provide in the alternative text is a replacement for the image, for someone who can't see it.

I think in this case what I would recommend is to not have any alternative text and just describe the image so that everyone can see the description. Something like, "when the user long-presses the space bar, they are able to drag the cursor to an arbitrary position. The floating cursor snaps to lines as the user moves it vertically, but does not snap to positions between characters horizontally. The floating cursor renders over the text field, while a faint shadow of the cursor is rendered in the text field in the location between characters where the cursor will drop into when released. The shadow cursor is a faint vertical bar with a blob at its bottom, while the floating cursor has the same appearance as a normal cursor (a blue vertical bar)."